blogPosts.json

```diff
@@ -0,0 +1,27 @@
+[
+    {
+        "post-title": "this is a post for blog",
+        "date": "Today",
+        "info": "Lorem ipsum dolor sit amet, consectetur adipiscing elit. In ut rutrum ipsum. Quisque mollis lectus at semper pulvinar. Cras at velit id leo semper lacinia. Suspendisse sit amet porttitor urna. Nulla commodo pharetra diam sed imperdiet. Vivamus rutrum, ex vitae ullamcorper maximus, libero felis placerat libero, eu faucibus nisl libero finibus dolor. Ut tempus, mauris nec ultricies mollis, velit augue fringilla quam, at suscipit ipsum tellus vitae nunc. Nunc consectetur libero non ultricies rutrum. Morbi elit erat, vulputate et libero tempor, posuere viverra arcu. "
+    },
+    {
+        "post-title": "this is a post for blog",
+        "date": "Tomorrow",
+        "info": "Lorem ipsum dolor sit amet, consectetur adipiscing elit. In ut rutrum ipsum. Quisque mollis lectus at semper pulvinar. Cras at velit id leo semper lacinia. Suspendisse sit amet porttitor urna. Nulla commodo pharetra diam sed imperdiet. Vivamus rutrum, ex vitae ullamcorper maximus, libero felis placerat libero, eu faucibus nisl libero finibus dolor. Ut tempus, mauris nec ultricies mollis, velit augue fringilla quam, at suscipit ipsum tellus vitae nunc. Nunc consectetur libero non ultricies rutrum. Morbi elit erat, vulputate et libero tempor, posuere viverra arcu. "
+    },
+    {
+        "post-title": "this is a post for blog",
+        "date": "Tuesday",
+        "info": "Lorem ipsum dolor sit amet, consectetur adipiscing elit. In ut rutrum ipsum. Quisque mollis lectus at semper pulvinar. Cras at velit id leo semper lacinia. Suspendisse sit amet porttitor urna. Nulla commodo pharetra diam sed imperdiet. Vivamus rutrum, ex vitae ullamcorper maximus, libero felis placerat libero, eu faucibus nisl libero finibus dolor. Ut tempus, mauris nec ultricies mollis, velit augue fringilla quam, at suscipit ipsum tellus vitae nunc. Nunc consectetur libero non ultricies rutrum. Morbi elit erat, vulputate et libero tempor, posuere viverra arcu. "
+    },
+    {
+        "post-title": "this is a post for blog",
+        "date": "12/31/1999",
+        "info": "Lorem ipsum dolor sit amet, consectetur adipiscing elit. In ut rutrum ipsum. Quisque mollis lectus at semper pulvinar. Cras at velit id leo semper lacinia. Suspendisse sit amet porttitor urna. Nulla commodo pharetra diam sed imperdiet. Vivamus rutrum, ex vitae ullamcorper maximus, libero felis placerat libero, eu faucibus nisl libero finibus dolor. Ut tempus, mauris nec ultricies mollis, velit augue fringilla quam, at suscipit ipsum tellus vitae nunc. Nunc consectetur libero non ultricies rutrum. Morbi elit erat, vulputate et libero tempor, posuere viverra arcu. "
+    },
+    {
+        "post-title": "new blog post",
+        "date": "Someone 2020",
```

Comment:
Is there no set format for the date displayed?

Reply:
it's just an h3 so not really